openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -283,7 +283,7 @@ def get_cmd_hazardous(*args, scope: $openc3_scope, token: $openc3_token, **kwarg
         item['states'].each do |name, hash|
           # To be hazardous the state must be marked hazardous
           # Check if either the state name or value matches the param passed
-          if hash['hazardous'] && (name == params[item['name']] || hash['value'].to_f == params[item['name']].to_f)
+          if hash['hazardous'] && (name == params[item['name']] || hash['value'] == params[item['name']])
```

Comment:
This bug resulted in treating the NOOP ascii command in our demo as hazardous because we call .to_f and the string value turns into 0.0. Not sure if it was there for a reason like floating point states or what?